README.md

```diff
@@ -0,0 +1,2 @@
+## Required tools
```

Comment:
This is a great change to make getting started with gocbcorex easier.  However it's not really related to the change at hand.  Would you be able to move this out to an independant commit?

Reply:
I didn't mean to actually open this yet... its still very WIP